bftengine/src/preprocessor/PreProcessor.hpp

```diff
@@ -32,7 +31,7 @@
 
 namespace preprocessor {
 
-struct ClientRequestState {
+struct RequestState {
```

Comment:
Nit - in the comment on the next line - the mutex is per request

Reply:
Will fix.